web/src/components/Change.res

```diff
@@ -305,6 +305,32 @@ module PinnedButton = {
   }
 }
 
+module MaskButton = {
+  @react.component
+  let make = (
+    ~change: SearchTypes.change,
+    ~status: MaskedChanges.changeStatus,
+    ~dispatchChange: MaskedChanges.dispatch,
+  ) => {
+    let (mask, unmask) = dispatchChange
+    let (tooltip, button, action) = switch status {
+    | Unmasked => (
+        "Mask change, it will not be shown on the board even if there is an update",
```

Comment:
What about ? "Mask this change. (The change will not be shown again on the board even if it got an update)"